src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,14 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball firstBall = new Ball();
+        Ball secondBall = new Ball();
+        Ball thirdBall = new Ball();
```

Comment:
The `Ball` objects are being instantiated here but are immediately overwritten in the loop. This is unnecessary and can be removed. Instead, directly assign the result of `lottery.getRandomBall()` to each element in the `balls` array.